package-drone/src/main/java/de/dentrassi/pm/jenkins/UploaderV3.java

```diff
@@ -84,7 +84,10 @@ public UploaderV3 ( final HttpClient client, final Run<?, ?> run, final TaskList
         catch ( final IOException e )
         {
             // delete in case of early abort
-            this.tempFile.delete ();
+            if ( ! this.tempFile.delete () )
+            {
+                listener.getLogger ().println ( "It should never be logged... but got the FindBug error fixed" );
```

Comment:
a better fix could be call `deleteOnExit()` on the same file, so it is marked to be deleted and a new tentative is performed when the JVM goes shutdown.